dockerfiles/hpcc/templates/_helpers.tpl

```diff
@@ -156,9 +156,9 @@ imagePullPolicy: {{ .root.Values.global.image.pullPolicy | default "IfNotPresent
 {{- end -}}
 
 {{/*
-A kludge to ensure host mounted storage (e.g. for minikube or docker for desktop) has correct permissions for PV
```

Comment:
Not sure about removing the word host here - AFAIK this is only relevant when you are using single-host systems and mapping your mount point directly to your host system

Reply:
or update the comment to cover the situation (?nfs) where it is required.

Reply:
yes, I should have updated the comment.
The NFS case requires forcePermissions=true.
This is the draft PR for the NFS example : https://github.com/hpcc-systems/HPCC-Platform/pull/13773